pulp_python/app/serializers.py

```diff
@@ -236,6 +236,13 @@ class Meta:
         )
         model = python_models.PythonPackageContent
 
+class JSONMultipleChoiceField(serializers.MultipleChoiceField):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def to_internal_value(self, data):
+        return list(super().to_internal_value(data))
```

Comment:
I don't like this hack I had to do to get the field to work with the model. Regular MultipleChoiceField did not work for the ArrayField model or JSONField model, but this hack allows this field to work with both. Need some advice on what to do here.